warcprox/main.py

```diff
@@ -106,6 +106,8 @@ def _build_arg_parser(prog=os.path.basename(sys.argv[0])):
     group = arg_parser.add_mutually_exclusive_group()
     group.add_argument('-j', '--dedup-db-file', dest='dedup_db_file',
             default='./warcprox.sqlite', help='persistent deduplication database file; empty string or /dev/null disables deduplication')
+    group.add_argument('--cdxserver-dedup', dest='cdxserver_dedup',
+            help='use a CDX Server for deduplication')
```

Comment:
Should this argument take a url parameter?

Reply:
I've made the help of this parameter a lot more explanatory. I think that this is want you wanted, right? There is no way to accept only type=URL parameter according to my knowledge.

Reply:
Oh, ok, I misunderstood at first, but the change you made is good.